tests/beaker_tests/cisco_bgp/test_bgp.rb

```diff
@@ -0,0 +1,260 @@
+    enforce_first_as:                       'default',
+    event_history_cli:                      'default',
+    event_history_detail:                   'default',
+    event_history_events:                   'default',
+    event_history_periodic:                 'default',
+    fast_external_fallover:                 'default',
+    flush_routes:                           'default',
+    graceful_restart:                       'default',
+    graceful_restart_helper:                'default',
+    graceful_restart_timers_restart:        'default',
+    graceful_restart_timers_stalepath_time: 'default',
+    isolate:                                'default',
+    log_neighbor_changes:                   'default',
+    maxas_limit:                            'default',
+    shutdown:                               'default',
+    suppress_fib_pending:                   'default',
+    timer_bestpath_limit:                   'default',
+    timer_bestpath_limit_always:            'default',
+    timer_bgp_holdtime:                     'default',
+    timer_bgp_keepalive:                    'default',
+  },
+  resource:       {
+    'bestpath_always_compare_med'            => 'false',
+    'bestpath_aspath_multipath_relax'        => 'false',
+    'bestpath_compare_routerid'              => 'false',
+    'bestpath_cost_community_ignore'         => 'false',
+    'bestpath_med_confed'                    => 'false',
+    'bestpath_med_missing_as_worst'          => 'false',
+    'bestpath_med_non_deterministic'         => 'false',
+    'disable_policy_batching'                => 'false',
+    'enforce_first_as'                       => 'true',
+    'event_history_cli'                      => 'size_small',
+    'event_history_detail'                   => 'size_disable',
+    'event_history_events'                   => 'size_small',
+    'event_history_periodic'                 => 'size_small',
+    'fast_external_fallover'                 => 'true',
+    'flush_routes'                           => 'false',
+    'graceful_restart'                       => 'true',
+    'graceful_restart_helper'                => 'false',
+    'graceful_restart_timers_restart'        => '120',
+    'graceful_restart_timers_stalepath_time' => '300',
+    'isolate'                                => 'false',
+    'log_neighbor_changes'                   => 'false',
+    'maxas_limit'                            => 'false',
+    'shutdown'                               => 'false',
+    'suppress_fib_pending'                   => 'false',
+    'timer_bestpath_limit'                   => '300',
+    'timer_bestpath_limit_always'            => 'false',
+    'timer_bgp_holdtime'                     => '180',
+    'timer_bgp_keepalive'                    => '60',
+  },
+}
+
+tests[:default_plat_1] = {
+  desc:           '1.2 Default Properties Platform-specific Part 1',
+  platform:       'n(3|9)k',
+  title_pattern:  "#{asn} default",
+  code:           [0, 2],
```

Comment:
If we are cleaning before the test then shouldn't this just be looking for 2?  Also, comes to mind that it's not clear from the docs or key name what this is referring to.  `puppet resource` command or something else?